src/MessagePack.GeneratorCore/CodeAnalysis/Definitions.cs

```diff
@@ -52,6 +52,8 @@ public bool IsStringKey
 
         public bool NeedsCastOnAfter { get; }
 
+        public bool HasInitOnlySetter { get; }
+
```

Comment:
Add HasInitOnlySetter property
it set true If there is even one init only setter.